wp-content/themes/core/assets/pcss/global/editor.pcss

```diff
@@ -32,6 +31,9 @@
 
 			@mixin t-body;
 			content: "Page Title";
```

Comment:
@GeoffDusome something to consider for a future update maybe. For localization reasons, can we hook into the page title and inject an attribute with a localized string? end result being `data-title="Localized Page Title"` and then use `content: attr(data-title);`

Reply:
Short answer: It looks like it's not currently possible to do this through the use of any hooks. I could of course always use JS to find the element and manually set the attribute, but this feels gross. My suggestion for sites that require localization - remove the pseudo element and update the padding to match top & bottom. I think it's obvious enough what the element is without the helper label.